common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -603,11 +601,10 @@ def oauth_params(self, custom_parameters, client_key, client_secret):
             except AttributeError:
                 self.user_username = ""
 
-        if self.open_in_a_new_page:
```

Comment:
Originally it seems like the LTI component never sent any username/email information. It seems like the original change came from the [following message](https://groups.google.com/forum/#!searchin/edx-code/LTI$20username/edx-code/VQDitIlQgnc/kooVNshqSboJ) in the edx-code list. It seems like someone had an external-only tool and wanted to provide it with user email/usernames. Their fix ONLY allowed for an external tool to get that information. We wanted it to be available for if you decided to have the tool appear as an iframe. In this way, it will still act exactly the same when you load an external tool, but it now also asks for the user information in the case of an iframe version. 

I'm not sure what you mean by "setting this to true" if you mean setting ```self.open_in_a_new_page``` to true, then the answer is because we don't want it to be a link to a new tab that takes the learner out of the edX experience of the course, we want it to be integrated within an iframe.